recording-oracle/src/modules/liquidity-score/liquidity-score.service.ts

```diff
@@ -90,6 +93,25 @@ export class LiquidityScoreService {
 
     const scoresFiles: UploadFile[] = [];
 
+    if (
+      !(
+        new Date(
+          startDate.getFullYear(),
+          startDate.getMonth(),
+          startDate.getDate(),
+        ) <= yesterday
+      )
+    ) {
+      this.logger.debug(
+        `Cannot calculate score now for ${campaign.address}, 24 hours have not passed yet since the last calculation.`,
+      );
+    }
+    if (startDate > campaign.endDate) {
```

Comment:
```suggestion
    if (startDate >= campaign.endDate) {
```

Did you add this just for the logs?

Reply:
Yes this is only for logs and with no equal as if it is equal the condition on the while loop will be true and the log will be not needed.

Reply:
In the loop it has no equal `startDate < campaign.endDate`